src/cfnlint/core.py

```diff
@@ -58,7 +63,7 @@ def run_cli(filename, template, rules, regions, override_spec, build_graph, regi
                     with open(os.path.join(path, f)) as schema:
                         REGISTRY_SCHEMAS.append(json.load(schema))
 
-    return run_checks(filename, template, rules, regions, mandatory_rules)
+    return run_checks(filename, template, rules, regions, validate_registry_types, mandatory_rules)
```

Comment:
We should add this to the and set a default value for backwards compatibility with people doing integration.
https://github.com/aws-cloudformation/cfn-lint/blob/main/docs/getting_started/integration.md.

Left as is we may break the integrations that some customers are doing.

This is why you will some functions have things like `mandatory_rules=None`.  Its all for backwards compatibility.